package try_Catch;

public class tryCatchEx {
    public static void main(String[] args) {
        
        try{
            int[] a = {4,5,1};
        System.out.println(a[2]);
        } catch(Exception e){
            System.out.print("Exception");
        }
    }

}
